src/clr-angular/data/datagrid/datagrid-column-toggle.ts

```diff
@@ -106,5 +117,22 @@ export class ClrDatagridColumnToggle {
 
   toggleSwitchPanel() {
     this.open = !this.open;
+    if (this.open && isPlatformBrowser(this.platformId)) {
+      this.zone.runOutsideAngular(() => {
```

Comment:
Not necessarily something we should do in this PR, but seems like we'll be doing this a number of times. It might be good to extract this into a service so we do it the same way each time with the same checks.

Reply:
Not a bad idea. Let's adopt the "Rule or three" principle :) When we do it for the 3rd time, we extract it into a helper class or service.